src/domain/Locales.ts

```diff
@@ -0,0 +1,4 @@
+export enum Locales {
```

Comment:
I think this might be a leftover that can be deleted

Reply:
Yep leftover, but we can use it for setting the locale in the action of the English banners so lets leave it for now